src/main/webapp/resources/js/components/lists/BasicList.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * Component to render a basic list with just a
+ * title and description for each item in the list
+ */
+
+import React from "react";
+import PropTypes from "prop-types";
+import { List, Typography } from "antd";
+
+const { Text } = Typography;
+
+export function BasicList({
+  itemLayout = "horizontal",
+  dataSource = { dataSource },
+  ...props
```

Comment:
Make sure you do something with these if you want them.

Reply:
Removed in 48b1a7b8bb8e39cb2a7eda2d963f27f207aae066